dandi/dandiapi.py

```diff
@@ -318,11 +318,17 @@ def __init__(
         self, api_url: Optional[str] = None, token: Optional[str] = None
     ) -> None:
         """
-        Construct a client instance from a server's base API URL and an
-        optional authentication token/API key.  If no URL is supplied, the URL
-        of the instance named in the :envvar:`DANDI_INSTANCE` environment
-        variable (default value: ``dandi``) is used.
+        Construct a client instance.
+
+        :param str api_url: API URL.
+        - For DANDI production, use  ``"https://api.dandiarchive.org/api"``
+        - For DANDI staging, use ``"https://api-staging.dandiarchive.org/api"``
+        - If no URL is supplied, the value of the  :envvar:`DANDI_INSTANCE` environment
+        variable (or default ``"dandi"``) is passed into ``for_dandi_instance()``.
```

Comment:
"is passed into ``for_dandi_instance()``" — I don't like this, as it's technically inaccurate.  How about "is used as the name of the API instance à la `for_dandi_instance()`"?